src/data/thunks.js

```diff
@@ -29,3 +29,20 @@ export function fetchCourseDetail(courseId) {
     }
   };
 }
+
+export function fetchWaffleFlags(courseId) {
+  return async (dispatch) => {
+    dispatch(updateStatus({ courseId, status: RequestStatus.IN_PROGRESS }));
+
+    try {
+      const waffleFlags = await getWaffleFlags(courseId);
+      dispatch(updateStatus({ courseId, status: RequestStatus.SUCCESSFUL }));
+      dispatch(fetchWaffleFlagsSuccess({ waffleFlags }));
+    } catch (error) {
+      // If fetching the waffle flags is unsuccessful,
+      // the pages will still be accessible and display without any issues.
+      // eslint-disable-next-line no-console
+      console.error({ courseId, status: RequestStatus.NOT_FOUND });
```

Comment:
Can you please remove the console log?  First because it should only ever be used for debugging.

Second, we probably shouldn't treat this API endpoint as optional, so we should probably just remove the `try..catch`.

Reply:
[inform] Implemented in follow-up commit: [`5f19746` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/5f19746fd6b2a803200abd8abdaac6adb5813fb2#diff-f708120055e193e6e1c566fefb57a208b514e2fd0f8944fc91ee1f0c97b0cd34R37)